xbmc/games/addons/input/GameClientInput.cpp

```diff
@@ -539,6 +540,7 @@ bool CGameClientInput::OpenMouse(const ControllerPtr& controller,
   if (bSuccess)
   {
     m_mouse = std::make_unique<CGameClientMouse>(m_gameClient, controller->ID(), mouse.get());
+    m_mouse->SetSource(std::move(mouse));
```

Comment:
GCC: `xbmc/games/addons/input/GameClientInput.cpp:543:33: warning: redundant move in initialization [-Wredundant-move]`
clang-tidy: `xbmc/games/addons/input/GameClientInput.cpp:543:24: warning: std::move of the const variable 'mouse' has no effect; remove std::move() or make the variable non-const [performance-move-const-arg]`